docs/management_api.md

```diff
@@ -34,6 +34,20 @@ This API follows the [ManagementAPIsService.RegisterModel](../frontend/server/sr
 ```bash
 curl -X POST  "http://localhost:8081/models?url=https://torchserve.pytorch.org/mar_files/squeezenet1_1.mar"
 
+{
+  "status": "Model \"squeezenet_v1.1\" Version: 1.0 registered with 0 initial workers. Use scale workers API to add workers for the model."
+}
+```
+Your model might be encrypted via [S3 SSE-KMS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingKMSEncryption.html). In this case, you need specify the following [environment variables](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html). 
+* AWS_ACCESS_KEY_ID
```

Comment:
Can we add more steps here? For example I configured an EC2 instance with the online console and am now following along.

It seems like we need to setup the Java AWS SDK, setup some local credentials and then export them? Would be useful to understand what needs to be run locally vs remotely and just have the list of instructions people need to input so they can follow along

Reply:
Yes, this implementation is using HTTP protocol, instead of S3. So AWS SDK is not needed. I will add steps for torchseve running on EC2 instance and local.